openbsd/Platform.c

```diff
@@ -368,16 +368,16 @@ void Platform_getBattery(double* percent, ACPresence* isOnAC) {
 
    *percent = NAN;
    if (found) {
-      /* last full capacity */
-      mib[3] = 7;
-      mib[4] = 0;
+      /* See "sys/dev/acpi/acpibat.c" of OpenBSD source code for the indexes
+         of the last field. */
```

Comment:
```suggestion
      /* See "sys/dev/acpi/acpibat.c" of OpenBSD source code for the indices of the last field. */
```

Reply:
Not a typo, just a different spelling of the word. You still want me to change it?

Reply:
AFAIU, indexes is mostly used for written documents, while indices is more often used in the mathematical sense. So yes, in this context "indices" is IMHO the preferred spelling¹.

¹ IANAL²
² Linguist

Reply:
@BenBE I decided to go with "indices" not for your reasoning, but because I didn't find any use of "indexes" in htop codebase but I got one result of "indices" when I ran `grep -r indices`.